src/extensions/workspace/workspace.provider.ts

```diff
@@ -22,7 +22,7 @@ export type WorkspaceConfig = {
 export default async function provideWorkspace(config: WorkspaceConfig, [scope, component, capsule]: WorkspaceDeps) {
   // This is wrapped since there are cases when there is no workspace, or something in the workspace is invalid
   // Those will be handled later
-  const consumer = await loadConsumerIfExist();
+  const consumer = await loadConsumer();
   if (consumer) {
```

Comment:
@qballer when using `loadConsumer` we know the consumer exists, so no need to check it the next line.

Reply:
@qballer , I remove this change. with this change, you can't run `bit init`!